drift/lib/src/runtime/query_builder/expressions/null_check.dart

```diff
@@ -29,6 +29,22 @@ Expression<T> coalesce<T extends Object>(List<Expression<T>> expressions) {
   return FunctionCallExpression<T>('COALESCE', expressions);
 }
 
+/// The ifnull() function returns a copy of its first non-NULL argument, or
+/// NULL if both arguments are NULL
```

Comment:
I think the Dartdoc for `ifnull` and `coalesce` should reference each other to highlight that a more general case with more arguments exists as well.